crates/vm/levm/src/vm.rs

```diff
@@ -592,37 +597,50 @@ impl VM {
     }
 
     /// Gets the max blob gas cost for a transaction that a user is willing to pay.
-    fn get_max_blob_gas_cost(&self) -> Result<U256, VMError> {
-        let blob_gas_used = U256::from(self.env.tx_blob_hashes.len())
+    fn get_max_blob_gas_price(&self) -> Result<U256, VMError> {
+        let blob_gasses = self.env.tx_blob_hashes.len();
+
+        let blob_gas_used: u64 = blob_gasses
+            .try_into()
+            .map_err(|_| VMError::Internal(InternalError::ConversionError))?;
```

Comment:
Can we change these names being used? Just leaving only one `let blob_gas_used` will be fine. If you want to can rename the blob hashes len to `blob_hashes_len` or something similar, but not `blob_gasses`

Reply:
Changed here: https://github.com/lambdaclass/ethrex/pull/1528/commits/1e5bf00c8712e96da3ca53bff5ebfdcfb0879dcf